### PR TITLE
[TASK] Drop support for PHP 7.0

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "~7.0 || ~7.1 || ~7.2 || ~7.3 || ~7.4",
+        "php": "~7.1 || ~7.2 || ~7.3 || ~7.4",
```
Comment (JakeQZ) [inline composer.json:40]: Noting that [the `.0` point version suffix](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) should be restored at some point so the behaviour is actually as intended (though there will not be a PHP 7.5).

Comment (oliverklee) [reply composer.json:40]: Agreed.

**Comment by JakeQZ** [comment]
The CI tests seemed to get stuck waiting for Lint and unit tests for PHP 7.0, but that's now resolved - ?

File: composer.json (hunk 1 above)
Comment (oliverklee) [reply composer.json:40]: #881

**Comment by oliverklee** [comment]
They still seem to get executed in #881. I have no idea why.

**Comment by JakeQZ** [comment]
> They still seem to get executed in #881. I have no idea why.

Something being cached?  Or is the any relevant GitHub configuratoin for the project that is external to the files visible within the source tree?

**Comment by JakeQZ** [comment]
Maybe this: https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157

**Pull request merged**